common/shlibs

```diff
@@ -1068,7 +1068,7 @@ libspectre.so.1 libspectre-0.2.6_1
 libsratom-0.so.0 sratom-0.4.0_1
 libpwquality.so.1 libpwquality-1.1.1_1
 libiml.so.0 iml-1.0.5_1
-libisl.so.23 isl-0.23_1
+libisl.so.23 isl-0.25_1
```

Comment:
this (and the cloog revbump) shouldn't be necessary unless they changed the abi without changing the soname.

Reply:
Ok, I try without.